tests/features/api_revisions.feature

```diff
@@ -0,0 +1,33 @@
+Feature: Revisions API
+
+    Scenario: Search revisions on file name 
+        Given user "<user>" is authenticated
+        And the Yoda revision API is queried with "<filename>"
+	    Then the response status code is "200"
+	    And "<revision_search_result>" is found 
+
+        Examples:
+        | user       | filename      | revision_search_result |
+	    | researcher | SIPI          | SIPI_Jelly_Beans       |
+
+
+    Scenario: Find actual revisions for one perticular dataobject
+        Given user "<user>" is authenticated
+        Given the Yoda revision API is queried with "<path>"
+	    Then the response status code is "200"
+	    And revisions list is found 
+
+        Examples:
+        | user       | path                                                                  |
+        | researcher | /tempZone/home/research-initial/testdata/SIPI_Jelly_Beans_4.1.07.tiff |
+
+
+    Scenario: Restore a revision 
```

Comment:
This scenario fails because the revision id changes.
Maybe we can add a given step 'And revision exists in "<path>"' that retrieves the revision id of an existing revision. Then in the API request step this revision id can be used. 

For examples see datarequest API test: https://github.com/UtrechtUniversity/irods-ruleset-uu/blob/4777f16238ff8377d5e0b2c73bc1baa38acb7850/tests/features/api_datarequest.feature#L15